core/src/main/java/org/apache/calcite/sql/validate/SqlValidatorImpl.java

```diff
@@ -449,7 +449,7 @@ private boolean expandSelectItem(
     // Expand the select item: fully-qualify columns, and convert
     // parentheses-free functions such as LOCALTIME into explicit function
     // calls.
-    SqlNode expanded = expand(selectItem, scope);
+    SqlNode expanded = expandSelectExpr(selectItem, scope, select);
```

Comment:
Not quite sure, with this change, shall we update the doc of `expandSelectItem ` ?

Reply:
Not necessary.